Python/Product/Analysis/ProjectEntry.cs

```diff
@@ -181,12 +179,14 @@ private class ActivePythonParse : IPythonParse {
         }
 
         internal void RaiseOnNewAnalysis() {
+            IsComplete = true;
             OnNewAnalysis?.Invoke(this, EventArgs.Empty);
         }
 
         public int AnalysisVersion { get; private set; }
 
         public bool IsAnalyzed => Analysis != null;
+        internal bool IsComplete { get; private set; }
```

Comment:
As mentioned elsewhere, I don't think this is valuable. If there is an available Analysis, we should treat it as complete (because it's the previous complete one, or the best available)

Reply:
This basically to give chance to analysis to complete. In VSC there is no way to re-trigger completion session or augment the list later, so the list should be there on the first try.  None of this solves the problem completely, but improves the experience a bit.